src/content/app-platform/overview/index.md

```diff
@@ -50,32 +50,22 @@ Custom Resources, some created by us, and others created by you. Together, they
 the desired state of the App Platform.
 
 For example, this "App" Custom Resource indicates that you want Kong installed
-on a specific workload cluster.
+on a specific workload cluster. We [default]({{< relref "app-platform/defaulting-validation/#defaulting" >}})
+some additional values for the workload cluster you select.
```

Comment:
We recently retired the `1.0.0` value for the `app-operator.giantswarm.io/version` label that was used for Helm 2.

We now always default this label and its value. I've also removed the other fields that will be defaulted and linked to the docs for that.